src/pages/informational/WpPlugin.tsx

```diff
@@ -0,0 +1,225 @@
+import character from "assets/laira/laira-waiving.png";
+import Copier from "components/Copier";
+import Image from "components/Image";
+
+export default function WpPlugin() {
+  return (
+    <div
+      className={`padded-container mt-16 grid content-start pb-16 @container`}
+    >
+      <h1 className="text-center @6xl:text-left text-4xl @6xl:text-5xl @6xl:leading-tight text-pretty mb-4 text-navy-d4">
+        Better Giving Wordpress plugin documentation
+      </h1>
+      <div className="py-5">
+        <h2 className="mb-4 col-span-full text-lg text-blue-d1 uppercase text-center">
+          Installing the WordPress Better Giving Donation Form Plugin
+        </h2>
+        <ol className="list-decimal">
+          <li>
+            Download the Better Giving{" "}
+            <a
+              href="https://github.com/AngelProtocolFinance/bg-donation-plugins/releases/download/v0.2/wordpress.zip"
+              target="_blank"
+              className="text-blue hover:text-blue-l2"
+              rel="noreferrer"
+            >
+              Wordpress Plugin
+            </a>{" "}
+            (ZIP file).
+            <p className="text-gray">
+              SHA256 Checksum:
+              34f1aa11ab99e691e290b128d11d019cc9f7cb29f14f01f921b03835316880ce
+            </p>
+          </li>
+          <li>
+            Install the plugin by uploading the ZIP file to your Wordpress
+            server, via the WP-Admin interface.
+          </li>
+          <li>Activate the Better Giving Donation Form plugin</li>
+        </ol>
+      </div>
+      <div className="py-5">
+        <h2 className="mb-4 col-span-full text-lg text-blue-d1 uppercase text-center">
+          Using and configuring Better Giving Donation Form Shortcode
+        </h2>
+        <div>
+          The shortcode used to insert a Better Giving donation form into your
+          wordpress blocks, pages and/or posts is <em>bg_donation_form</em>.
+        </div>
+        <div className="py-2">
+          At a minimum, the following MUST be provided for the shortcode to work
+          correctly and render the donation form: a Nonprofit Account ID. This
+          is the number that was assigned to your nonprofit when you signed up
+          with Better Giving and is unique to it (ex. 121).
+        </div>
+        <div className="mx-auto flex flex-row max-w-md gap-4 bg-blue-l2 p-4 my-4 rounded">
```

Comment:
`display:flex` is `flex-row` by default
```suggestion
        <div className="mx-auto flex max-w-md gap-4 bg-blue-l2 p-4 my-4 rounded">
```